Automattic/woocommerce-payments
# Remove duplicative multi-currency task

Fixes #3129 

#### Changes proposed in this Pull Request

Remove multi-currency task keeping its setup page

As mentioned in the linked issue, we have an item saying "Sell worldwide in multiple currencies" doing the same. One is a task in the overview tasks list section, and the other is an inbox note on the same page. This PR removes the task in favor of the inbox note.

* Create a new page to handle the multi-currency setup
* Remove the task displayed on the overview page that pointed to the multi-currency setup
* Point the URLs that used to go to the task's page to the newly added page
* Remove the tests for the task functionalities

![image](https://user-images.githubusercontent.com/41606954/143688167-232feef3-5ad7-442a-acf8-9a6d933f56c6.png)

#### Testing instructions

* Enable the `_wcpay_feature_account_overview_task_list` feature (there's an option in the WCPay Dev tools)
* Go to Payments -> Overview
* Under the section "Things to do", make sure that the item "Sell worldwide in multiple currencies" doesn't show up. This section won't show up if there are no tasks to display. You can make it show up by creating some disputes
* At the bottom of that page, look for the note titled "Sell worldwide in multiple currencies". Make sure that "Set up now" takes you to `?page=wc-admin&path=/payments/multi-currency-setup` and not to `?page=wc-admin&task=woocommerce-payments--multi-currency-setup`. I had to disable and enable again WCPay for this to change

-------------------


**Post merge**
